auto_add_names.py

```diff
@@ -0,0 +1,35 @@
+## This is a small script to automatically add your name in the list alphabetically
+
+import re
+import os
+
+##part 1- Get the info
+print('Welcome Women Githuber')
+name=input('Enter the name you wish to see on the list::')
```

Comment:
http://stackoverflow.com/questions/7450223/how-do-i-get-user-input-from-the-keyboard-in-python-2